framework/infrastructure/gcp/compute.py

```diff
@@ -558,6 +566,60 @@ def get_backend_service_backend_health(self, backend_service, backend):
             .execute()
         )
 
+    def create_serverless_neg(self, name: str, region: str, service_name: str):
```

Comment:
The pattern here is `<verb>_<resource_type>_<resurce_subtype_if_any>`. For example, we have 
* create_url_map
* create_url_map_with_content.

We should rename `create_serverless_neg` to `create_neg_serverless`

Reply:
You missed `delete_serverless_neg` -> `delete_neg_serverless`